src/twisted/python/test/test_win32.py

```diff
@@ -34,3 +34,41 @@ def test_emptyStringArg(self):
         quoted empty string.
         """
         self.assertEqual(win32.cmdLineQuote(""), '""')
+
+
+class DeprecationTests(unittest.TestCase):
+    """
+    Tests for deprecated (Fake)WindowsError.
+    """
+
+    def test_deprecation_FakeWindowsError(self):
+        """Importing C{FakeWindowsError} should trigger a L{DeprecationWarning}."""
+
+        def import_FakeWindowsError():
+            from twisted.python.win32 import FakeWindowsError
+
+            FakeWindowsError  # pretend to use, for flake8
```

Comment:
An alternative to consider might be `namedAny("twisted.python.win32.FakeWindowsError")`.  This avoids binding a name so avoids needing a flake8 work-around.